### ISSUE TITLE
Design Intent - Accessibility Feedback - [Reapers Team, VA.gov Federated Portal]

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#129256]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket any time** after you've finished reviewing it. You don't need to confirm with Governance before closing.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Consider:** Consider the placement of indicated actions (e.g.  below the Next/Previous primary/secondary process/form actions. This type of organization, where resources necessary for the user to complete are below a primary action, can be confusing for some assistive technology users such as screen reader users. As sighted users, we can often see the context due to visual proximity, but for some users, the next/previous buttons indicate the end of that section, so including relevant actions below those elements can feel disjointed.
- [ ] **Should:** Update copy to avoid the use of directional language and the lack of specificity, or difference in meaning to different users, of "bottom of this web page". Many users may interpret that to mean the bottom of the web page, i.e. the footer. For assistive technology users this may be even more confusing. Aligned with my first feedback item, think about page organization, and pulling relevant content into the step itself, so that directional language can be completed avoided. 

> "Your next step is to watch and confirm completion of the Orientation Video, which can be found in the "Orientation Tools and Resources" link at the bottom of this web page." 

- [ ] **Consider** On the selection for Telecounseling or in-person appointment, there is a conditional "Additional information" block. Consider the disconnect between the first selection triggering the conditional. Typically, when there is a conditional, it is the last radio button (see VADS). 
- [ ] **Should:** On the selection for Telecounseling or in-person appointment, having two submit actions on the same page may be confusing for users, who are looking for a primary action. 
 

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
- [ ] Add the 'collaboration-cycle-mobile-app' label if this feedback is for the mobile app
